Source/Core/DolphinWX/ControllerConfigDiag.cpp

```diff
@@ -227,7 +227,7 @@ wxSizer* ControllerConfigDiag::CreateWiimoteConfigSizer()
 
   auto* const box = new wxStaticBoxSizer(wxVERTICAL, this, _("Wii Remotes"));
 
-  m_passthrough_bt_radio = new wxRadioButton(this, wxID_ANY, _("Passthrough a Bluetooth adapter"),
+  m_passthrough_bt_radio = new wxRadioButton(this, wxID_ANY, _("Passthrough a Bluetooth Adapter"),
```

Comment:
If you change this, you need to change "Emulate the Wii's Bluetooth adapter" below too.